Directory.Build.props

```diff
@@ -0,0 +1,45 @@
+<Project>
+
+  <PropertyGroup>
+    <VersionPrefix>1.2.0</VersionPrefix>
+    <LangVersion>11.0</LangVersion>
+    <Nullable>warnings</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <NoWarn>$(NoWarn);1591;1998;NU5105</NoWarn>
+    <DebugType>embedded</DebugType>
+    <GitHubOrganization>FacilityApi</GitHubOrganization>
+    <RepositoryName>FacilityLanguageServer</RepositoryName>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/$(GitHubOrganization)/$(RepositoryName)</PackageProjectUrl>
+    <PackageReleaseNotes>https://github.com/$(GitHubOrganization)/$(RepositoryName)/blob/master/ReleaseNotes.md</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/$(GitHubOrganization)/$(RepositoryName).git</RepositoryUrl>
+    <Authors>Ed Ball, Dave Dunkin</Authors>
+    <Copyright>Copyright $(Authors)</Copyright>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>false</IsTestProject>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(BuildNumber)' != '' ">
+    <AssemblyVersion>$(VersionPrefix).$(BuildNumber)</AssemblyVersion>
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <FacilityVersion>2.11.0</FacilityVersion>
```

Comment:
Does this address https://github.com/FacilityApi/FacilityLanguageServer/issues/3? And does https://github.com/FacilityApi/FacilityLanguageServer/blob/c00c175c6bce8a14eb8a743574549b1b61f89002/src/Facility.LanguageServer/Facility.LanguageServer.csproj#L9 need to be updated as well?

Reply:
I copied this file from FacilityCSharp. That PackageReference would need to be updated to use the FacilityVersion property.